packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -329,19 +326,14 @@ export const validateSummary = (
 	return { ids, redirectTable };
 };
 
-for (const enableBlobPlaceholdersFlag of [false, true]) {
-	describe(`BlobManager (blob placeholders: ${enableBlobPlaceholdersFlag})`, () => {
-		const handlePs: Promise<IFluidHandle<ArrayBufferLike>>[] = [];
```

Comment:
The handlePs weren't actually getting used.  Removed to make it more obvious that we're ignoring their outcomes (generally, we seem to be expecting to watch the progress by inspecting the internals/mocks and detect failure that way rather than watching the handle).